internal/serverinstall/serverinstall.go

```diff
@@ -100,6 +100,12 @@ type InstallRunnerOpts struct {
 	AdvertiseClient *serverconfig.Client
 }
 
+// An optional interface that the installer can implement to request
+// an ondemand runner be registered.
+type OndemandRunnerConfig interface {
+	OndemandRunner() *pb.OndemandRunner
+}
```

Comment:
Am I missing something in the PR or is this not implemented yet?

Reply:
eh? It's implemented, it's used by internal/cli/install.go